uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -26,7 +26,10 @@ function hrcore_civicrm_config(&$config) {
  * @param array $tasks
  */
 function hrcore_civicrm_searchTasks($objectName, &$tasks) {
-  if ($objectName !== 'contact' || !CRM_Core_Permission::check('create users')) {
+  $isContact = $objectName === 'contact';
```

Comment:
What is the problem with adding functions to hrcore? I feel like it's still better than adding the logic directly to the hook itself.

In the future, if another dev needs to add a new search task, he'll find this hook check how you did things and will probably think that is fine to just add some other if or else and add the tasks to the list. Suddenly, this hook will have [more than 100 lines](https://github.com/civicrm/civihr/blob/staging/hrui/hrui.php#L227-L327) and no one will ever refactor it